frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -90,4 +190,54 @@ describe("scenarios > dashboard > chained filter", () => {
       });
     });
   }
+
+  context("reproduces metabase#13868", () => {
+    it.only("can use a chained filter with embedded SQL questions", () => {
+      createDashboardWithQuestion({}, dashboardId => {
+        cy.visit(`/dashboard/${dashboardId}`);
+        cy.request("PUT", `/api/dashboard/${dashboardId}`, {
+          embedding_params: {
+            city: "enabled",
+            state: "enabled",
+          },
+          enable_embedding: true,
+        });
+      });
+      // First make sure normal filtering works - we reuse the chained filter test above
+      // Select Alaska as a state. We should see Anchorage as a option but not Anacoco
+      cy.findByText("State").click();
+      popover().within(() => {
+        cy.findByText("AK").click();
+        cy.findByText("Add filter").click();
+      });
+      cy.findByText("City").click();
+      popover().within(() => {
+        cy.findByPlaceholderText("Search by City").type("An");
+        cy.findByText("Anacoco").should("not.exist");
+        cy.findByText("Anchorage").click();
+        cy.findByText("Add filter").click();
+      });
+      cy.contains("Count");
```

Comment:
```suggestion
      cy.get(".y-label").contains("Count");
```

Contains works on a word in a sentence, as well. See this example:
![image](https://user-images.githubusercontent.com/31325167/102140566-b4bc8100-3e5f-11eb-89d4-497a9a32217f.png)

I think we're safer if we narrow it down to a `.y-label`.